tree/ntupleutil/CMakeLists.txt

```diff
@@ -34,4 +34,8 @@ if(MSVC)
   target_compile_definitions(ROOTNTupleUtil PRIVATE _USE_MATH_DEFINES)
 endif()
 
+if (CMAKE_COMPILER_IS_GNUCC AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 9.0)
+    target_link_libraries(ROOTNTupleUtils stdc++fs)
+endif()
+
```

Comment:
Hm, for me this would be a strong reason not to use `std::filesystem` in the interface definition, especially if we don't need it...

Reply:
this was removed.